Microsoft/helium
# Refactor app insights usage

Moving our use of App Insights into a wrapper. Using the ServiceLocator framework to supply the instrumentation key via KeyVault.

To test:

1. Make sure there is a secret in KeyVault called `AppInsightsInstrumentationKey` with your instrumentation key as the value.
2. Build/Run the app
3. Make sure request data shows up in app insights
